components/global_header/right_controls/plan_upgrade_button/index.tsx

```diff
@@ -37,6 +37,8 @@ const PlanUpgradeButton = (): JSX.Element | null => {
     const {formatMessage} = useIntl();
 
     openPricingModal = useOpenPricingModal();
+    const openPricingModalA = useOpenPricingModal('PlanUpgradeButton_A');
+    const openPricingModalB = useOpenPricingModal('PlanUpgradeButton_B');
```

Comment:
I think the way opening the pricing modal works for playbooks and boards is they import `openPricingModal`. This  means that those elements won't get the variation. Should they? i.e. should we determine what the A/B test value is and assign to `openPricingModal` the result of that?

Reply:
I'm wondering if we need the variant on the track event at all - see my comment above